### PR TITLE
Integrate v0.70.0

### PR DESCRIPTION
**Pull request by chiaramooney**
### Type of Change
- Integration

### Why
Integrate 0.70-stable.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10516)

### CODE REVIEW

File: packages/@react-native-windows/tester/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "@types/node": "^14.14.22",
     "eslint": "^7.32.0",
     "just-scripts": "^1.3.3",
-    "react-native": "0.70.0-rc.4",
+    "react-native": "0.70.0",
```
Comment (vmoroz) [inline packages/@react-native-windows/tester/package.json:32]: Should we remove it? Or at least it must be the same version as above: "^0.70.0".

File: vnext/package.json [hunk 2]
```diff
@@ -76,22 +76,22 @@
     "metro-config": "^0.70.1",
     "prettier": "^2.4.1",
     "react": "18.1.0",
-    "react-native": "0.70.0-rc.4",
+    "react-native": "0.70.0",
```
Comment (vmoroz) [inline vnext/package.json:79]: >0.70.0 [](http://example.com/codeflow?start=21&length=6)

Should it be "^0.70.0"?

**Review by vmoroz — APPROVED** [review]
:shipit:

File: packages/@react-native-windows/tester/package.json (hunk 1 above)
Comment (chiaramooney) [reply packages/@react-native-windows/tester/package.json:32]: According to Nick's notes on the release process and what we've done historically, this dep stays just 0.70.0

**Pull request merged**